### PR TITLE
Main file + standardized code

### PR DESCRIPTION
**Pull request by miklimek**
Changes:
- combined all thermodynamics function in one file
- added user input validation
- fixed some typos in optics
- changed displayed text so that it's mostly the same in all functions
- stylistic changes (lines that separate different functions etc.)

Please check for any typos and comment if you have problems with changes that I've made

### CODE REVIEW

File: Physicist's_Toolbox.cpp [hunk 1]
```diff
@@ -1,16 +1,50 @@
...
-    return 0;
+	std::cout << "Welcome to Physicist's Toolbox!\n\n"
+		<< "Please remember to use International System of Units (SI)\n" 
+		<< "---------------------------------------------------------------------------\n"
+		<< "Which field of physics do you need help with today?\n\n"
+		<< " - elctrostatics(1)\n"
+		<< " - thermodynamics(2)\n"
+		<< " - gravitation(3)\n"
+		<< " - kinematics(4)\n\n"
+		<< "Field: ";
```
Comment (PaulinaGacek) [inline Physicist's_Toolbox.cpp]: Everything's ok but you have forgotten about optics and dynamics.

Comment (miklimek) [reply Physicist's_Toolbox.cpp]: You're viewing an older commit, I've added them as well. Don't know why it didn't show you the newest version

File: Physicist's_Toolbox.cpp [hunk 2]
```diff
@@ -1,16 +1,99 @@
-//Program będzie wykonywać obliczenia fizyczne za nas.
-// W końcowej fazie projektu pojawi się tutaj kod, na ten moment trwają pracę nad mniejszymi programikami z funkcjami.
 #include <iostream>
-#include <string>
+#include "declarations.h"
```
Comment (MStrzezon) [inline Physicist's_Toolbox.cpp:2]: Shouldn't you #include "*.cpp"? Because I can't run this application ( undefined reference to `electrostatics()')

Comment (miklimek) [reply Physicist's_Toolbox.cpp:2]: I'm pretty sure you just need to have both files open and it'll work just fine. There're declarations for every function in the declarations.h header, so if you open all necessary files at the same time it compiles

Comment (MStrzezon) [reply Physicist's_Toolbox.cpp:2]: I have both open and it doesn't work :-(  I used to be like that before and only including the .cpp files worked. Only in VS Code it works me

Comment (MStrzezon) [reply Physicist's_Toolbox.cpp:2]: However, it does not work 
image

Comment (miklimek) [reply Physicist's_Toolbox.cpp:2]: If you open the developer command prompt window and enter the command: cl /EHsc Physicist's_Toolbox.cpp MergedElectrostatics.cpp Optics.cpp gravitation.cpp kinematics.cpp dynamics.cpp thermodynamics.cpp
it should compile and then you can open the Physicist's_Toolbox.exe program and try to use it.

File: dynamics.cpp [hunk 3]
```diff
@@ -1,290 +1,307 @@
...
+        std::cout << "\nThe final speed is equal to: " << acceleration * time << " m/s\n";
     }
     else
     {
-        cout << "The number you chose does not correspond to any available function. Please try again.\n\n";
-        slippery_slope_without_friction();
-    }
-}
+        std::cout << "The number you've chosen does not correspond to any available function. Please try again\n"
+            << "\n---------------------------------------------------------------------------\n";
```
Comment (MStrzezon) [inline dynamics.cpp:75]: Is "}" missing here?

File: Physicist's_Toolbox.cpp (hunk 2 above)
Comment (MStrzezon) [reply Physicist's_Toolbox.cpp:2]: Okay, I will try.

File: dynamics.cpp (hunk 3 above)
Comment (MStrzezon) [reply dynamics.cpp:75]: After function

File: Physicist's_Toolbox.cpp (hunk 2 above)
Comment (MStrzezon) [reply Physicist's_Toolbox.cpp:2]: It still does not work for me, maybe if you merge it then I will try.

File: dynamics.cpp (hunk 3 above)
Comment (miklimek) [reply dynamics.cpp:75]: Yeah, you're right I'll fix it

**Comment by Julia-Pl1** [comment]
Everything looks fine to me, I'm pretty sure it's all correct.

**Review by Julia-Pl1 — APPROVED** [review]
Everything looks fine to me, i'm really content with your work. The code looks great and I agree with all of the changes, that you made.

File: Physicist's_Toolbox.cpp (hunk 2 above)
Comment (miklimek) [reply Physicist's_Toolbox.cpp:2]: I've added the include "*.cpp". Hopefully it works for you now

Comment (MStrzezon) [reply Physicist's_Toolbox.cpp:2]: Yeah, now it works.

**Pull request merged**